app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">ParkingSystem</string>
+    <string name="text_main_activity_title">Welcome to Parking System</string>
+    <string name="button_main_activity_select_park">Select your parking spot</string>
```

Comment:
The first bottom is to set the amount of parking availables, not an actual space.
please, correct strings, and ids related

Reply:
Done!